.travis.yml

```diff
@@ -3,32 +3,27 @@ before_install:
 - ls -l aws-android-sdk-appsync-tests/src/main/res/raw/awsconfiguration.json
 - wc aws-android-sdk-appsync-tests/src/main/res/raw/awsconfiguration.json
 - chmod +x gradlew
-- yes | sdkmanager "platforms;android-25"
-- yes | sdkmanager tools
-- sdkmanager "system-images;android-25;google_apis;armeabi-v7a"
+- yes | sdkmanager "platforms;android-27"
 - android list target
-- echo no | avdmanager create avd --force -n test -k "system-images;android-25;google_apis;armeabi-v7a"
-- $ANDROID_HOME/emulator/emulator -avd test -no-skin -no-audio -no-window &
+- echo no | android create avd --force -n test -t android-22 --abi armeabi-v7a
```

Comment:
Don't you have to change all of the values to the same thing? Line 6 says `android-27`, but this says `android-22`?

Reply:
Yeah...and the weird thing is that it's what it is currently in master. I'm just reverting the changes I made to this file for this PR and will pick that up separately.